.kitchen.vagrant.yml

```diff
@@ -0,0 +1,101 @@
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: ansible_playbook
+  hosts: localhost
+  roles_path: ../
+  require_ansible_repo: false
+  require_ansible_omnibus: false
+  require_ansible_source: false
+  require_pip: true
+  ansible_version: 2.4.3.0
+  http_proxy: <%= ENV['HTTP_PROXY'] %>
+  https_proxy: <%= ENV['HTTPS_PROXY'] %>
+  no_proxy: localhost,127.0.0.1
+  ignore_extensions_from_root: [".git",".idea",".kitchen.yml"]
+  ignore_paths_from_root: [".git",".idea",".kitchen"]
+  <% if ENV['VERSION'] %>
+  attributes:
+    extra_vars:
+      es_major_version: "<%= ENV['VERSION'] %>"
+      <% if ENV['VERSION'] == '5.x' %>
+      es_version: '5.6.7'
+      <% end %>
+  <% end %>
+
+verifier:
+  name: inspec
+  sudo: true
+  inspec_tests:
+    - git: https://github.com/elastic/elasticsearch-inspec.git
```

Comment:
I wasn't able to get this far when testing locally. However how will this work? This is currently a private repo so my credentials will somehow need to be forwarded to inspec to be able to pull this repo.

Reply:
You should have access to that repo soon for the verifier part (having someone add you now). It will eventually be public as well.

Reply:
I do actually already have access to this, I was mostly curious how this works from a test kitchen point of view. If it was using the git url (git@github.com:elastic/elasticsearch-inspec.git) then kitchen would be able to use my ssh agent to get the repo. However because it is using https it needs to have a GitHub access token configured somewhere. Is there some magic environment variable that needs to be exported so that test kitchen can use https auth somehow?